tests/core/fs/workspacefs/test_workspace_fs.py

```diff
@@ -7,7 +7,7 @@
 from trio import open_nursery
 
 from parsec.api.protocol import DeviceID, RealmID, RealmRole
-from parsec.api.data import BaseManifest as BaseRemoteManifest, EntryName
```

Comment:
Strange renaming, especially given this class is monkeypatched in the test (where TypeVar are typing-only concepts)

Reply:
should revert this rename

Reply:
Next comment the complete change, for me you were commenting a deleted line

Reply:
done